src/components/widgets/status/StatusTab.vue

```diff
@@ -422,17 +420,71 @@ export default class StatusTab extends Mixins(StateMixin, FilesMixin) {
 </script>
 
 <style lang="scss" scoped>
-  .print-thumb {
-    display: block;
-    max-height: 110px;
-  }
+.print-thumb {
+  display: block;
+  max-height: 110px;
+}
 
-  .filename {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    // width: 200px;
-    direction: rtl;
-    text-align: left;
-  }
+.filename {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  // width: 200px;
+  direction: rtl;
+  text-align: left;
+}
+
+.position-relative {
+  position: relative;
+}
+
+.progress-wrapper {
+  position: relative;
+  width: 90px;
+  /* Set the width and height to match progress circle size */
+  height: 90px;
+  /* Set the width and height to match progress circle size */
+  overflow: hidden;
+  border-radius: 50%;
+}
+
+.progress-circle {
+  position: absolute;
+  z-index: 1;
+}
+
+.thumbnail-background {
+  position: absolute;
+  width: 80%;
+  /* 10% less than the size of the progress circle */
+  height: 80%;
+  /* 10% less than the size of the progress circle */
+  border-radius: 50%;
+  background-color: rgba(0, 0, 0, 0.5);
+  /* Adjust transparency here */
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 0;
+}
+
+.print-thumb {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 0;
+  max-height: 70px;
+  /* Adjust as needed */
+  border-radius: 50%;
+  opacity: 0.8;
+  /* Adjust opacity as needed */
+}
+
+.percentage-wrapper {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-43%, 200%);
```

Comment:
@matmen  not to happy with this absolute positioning.. probably a better way to do this. Am sure @pedrolamas  when we gets to looking into this might have some other ideas. if I find a better way to do this without using numbers I will update it

Reply:
think i got it. added some rows. and center justify the content. looks clearer and less fiddly.